eiffel-vocabulary/EiffelSourceChangeCreatedEvent.md

```diff
@@ -0,0 +1,210 @@
+
+#### data.issues.uri
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The URI of the issue.
+
+#### data.issues.transition
+__Type:__ String  
+__Required:__ Yes  
+__Legal values:__ RESOLVED, PARTIAL, REMOVED  
+__Description:__ The new state of the issue: does the change resolve it, partially resolve it or remove it?
+
+### data.gitIdentifier
+__Type:__ Object  
+__Required:__ No  
+__Description:__ Identifier of a Git change.
+
+#### data.gitIdentifier.commitId
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The commit identity (hash) of the change.
+
+#### data.gitIdentifier.branch  
+__Type:__ String  
+__Required:__ No  
+__Description:__ The name of the branch where the change was made.
+
+#### data.gitIdentifier.repoName  
+__Type:__ String  
+__Required:__ No  
+__Description:__ The name of the repository containing the change.
+
+#### data.gitIdentifier.repoUri  
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The URI of the repository containing the change.
+
+### data.svnIdentifier
+__Type:__ Object  
+__Required:__ No  
+__Description:__ Identifier of a Subversion change.
+
+#### data.svnIdentifier.revision
+__Type:__ Integer  
+__Required:__ Yes  
+__Description:__ The revision of the change.
+
+#### data.svnIdentifier.directory  
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The directory (branch/tag) of the change.
+
+#### data.svnIdentifier.repoName  
+__Type:__ String  
+__Required:__ No  
+__Description:__ The name of the repository containing the change.
+
+#### data.svnIdentifier.repoUri  
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The URI of the repository containing the change.
+
+### data.ccCompositeIdentifier
+__Type:__ Object  
+__Required:__ No  
+__Description:__ Identifier of a composite ClearCase change – in other words, not single file commit, but analogous of repository-wide commits of e.g. SVN or Git.
+
+#### data.ccCompositeIdentifier.vob
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The ClearCase VOB name.
+
+#### data.ccCompositeIdentifier.branch
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The branch of the change.
+
+#### data.ccCompositeIdentifier.configSpec
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The URI of the relevant ClearCase config spec.
+
+### data.hgIdentifier
+__Type:__ Object  
+__Required:__ No  
+__Description:__ Identifier of a Mercurial change.
+
+#### data.hgIdentifier.commitId
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The commit identity (hash) of the change.
+
+#### data.hgIdentifier.branch
+__Type:__ String  
+__Required:__ No  
+__Description:__ The branch of the change.
+
+#### data.hgIdentifier.repoName
+__Type:__ String  
+__Required:__ No  
+__Description:__ The name of the repo.
+
+#### data.hgIdentifier.repoUri
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The URI of the repo.
+
+### data.gerritIdentifier
```

Comment:
It's a balance, I guess. As you say, you could go conservative and treat it as any other Git repo, or you could make allowances for particular information (e.g. patch set, Change-Id) where convenient. As a rule I favor conservatism, so I think you have a point.